src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html

```diff
@@ -596,9 +596,9 @@ <h3>
 			die Plausibilität Ihres Anrufs zu überprüfen. Diese Fragen dienen der
 			Vorbeugung einer missbräuchlichen Infektionsmeldung und daraus
 			resultierender fehlerhafter Warnungen und Risikowerte. Nach ausreichender
-			Beantwortung dieser Fragen werden Sie nach Ihrer Smartphone-/Telefonnummer
+			Beantwortung dieser Fragen werden Sie nach Ihrer Handy-/Telefonnummer
```

Comment:
Why is it changed back to term "Handy"? Shouldn't it be "Smartphone" everywhere?

Reply:
Hi Conrad, after discussing with multiple colleagues, I think when you are referring to a phone number, you generally speak of Handynummer or Telefonnummer - you don't really ask for somebody's Smartphone-Nummer, so in this instance I decided to change it back.

Reply:
ok